src/v2/core.ts

```diff
@@ -80,12 +82,28 @@ export interface CloudEvent<T> {
   data: T;
 }
 
-/** A handler for CloudEvents. */
+/**
+ * A handler for CloudEvents.
```

Comment:
Is "CloudEvents" a literal?

Reply:
Kinda? It's the name of a class, but it's also the name of a specification (which is what I intended here). The capitalization is what is at https://cloudevents.io